media/js/mozorg/home/ga-tracking.js

```diff
@@ -5,6 +5,15 @@
 $(function () {
     'use strict';
 
+    var $mainNav;
+
+    //generic links
+    $mainNav = $('#nav-main-menu li a');
+    $mainNav.attr({
+        'data-element-location': 'nav click',
+        'data-link-type': $mainNav.data('name')
+    });
+
```

Comment:
Hm, working with an array of elements like this is a bit confusing to read. Let's change it to be a bit more explicit in intent. Something like:

```
$('#nav-main-menu li a').each(function() {
    var $this = $(this);
    $this.attr({
        'data-element-location': 'nav click',
        'data-link-type': $this.data('name')
    });
});
```